spec.html

```diff
@@ -44215,15 +44232,17 @@ <h1>
         <emu-alg>
           1. Let _view_ be _viewRecord_.[[Object]].
           1. Let _bufferByteLength_ be _viewRecord_.[[CachedBufferByteLength]].
-          1. Assert: IsDetachedBuffer(_view_.[[ViewedArrayBuffer]]) is *true* if and only if _bufferByteLength_ is ~detached~.
-          1. If _bufferByteLength_ is ~detached~, return *true*.
+          1. If IsDetachedBuffer(_view_.[[ViewedArrayBuffer]]) is *true*, then
+            1. Assert: _bufferByteLength_ is ~detached~.
```

Comment:
I'd prefer to undo this change. This assert is now a one-way implication of IsDetachedBuffer() -> _bufferByteLength_ is DETACHED. The original assert is a two-way implication.

Reply:
It's still two-way, and actually stronger from the new type assertion.
**before**:
* IsDetachedBuffer(_view_.[[ViewedArrayBuffer]]) is true ⟺ _bufferByteLength_ is DETACHED
* i.e., (A ∧ B) ∨ (¬A ∧ ¬B)

**after**:
* IsDetachedBuffer(_view_.[[ViewedArrayBuffer]]) is true ⇒ _bufferByteLength_ is DETACHED **_but also_** (_after the early return_) [IsDetachedBuffer(_view_.[[ViewedArrayBuffer]]) is not true ⇒] _bufferByteLength_ is a non-negative integer
* i.e., (A ⇒ B) ∧ (¬A ⇒ ¬B)

Reply:
Ah, right, there's the second assert. Thanks for correcting me.